Project-II-Calculator/calculator/src/components/BoilingWater.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import '../css/index.css';
+
+
+
+function BoilingVerdict(props) {
+  if (props.celsius >= 100) {
+    return <p>The water would boil.</p>;
+  } else if (props.celsius <= 0) {
+    return <p>The water would freeze.</p>;  
+  } else {
+    return <p>The water would not boil.</p>;  
+  }
+}
+
+class BoilingCalculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+    this.state = {temperature: ''};
```

Comment:
```suggestion
    this.state = {temperature: null};
```
Typically we initialize numbers in state as null if we don't have a starting number.